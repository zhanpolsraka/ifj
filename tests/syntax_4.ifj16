//2
class Main{
    static void run() {
        // chybi zavorky.
        int a = ifj16.readInt 1;
    }
}
