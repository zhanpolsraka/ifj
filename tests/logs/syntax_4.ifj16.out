
Syntax error! Unknown expression with token "1"!

